src/autoscaler/pruner/cmd/pruner/pruner_suite_test.go

```diff
@@ -66,16 +66,17 @@ func initConsul() {
 
 func initConfig() {
 	cfg.Logging.Level = "debug"
+	dbUrl := os.Getenv("DBURL")
```

Comment:
Perhaps I missed something... Pruner test cases can be run separately or before db package test cases. 

we can just add check before the following line and fail the test of $DBURL is not set
https://github.com/cloudfoundry-incubator/app-autoscaler/blob/develop/src/autoscaler/pruner/cmd/pruner/pruner_suite_test.go#L70